zng/net.go

```diff
@@ -13,21 +13,20 @@ func NewNet(s *net.IPNet) Value {
 	return Value{TypeNet, EncodeNet(s)}
 }
 
-func EncodeNet(subnet *net.IPNet) zcode.Bytes {
-	var b [32]byte
-	ip := subnet.IP.To4()
-	if ip != nil {
-		copy(b[:], ip)
+func AppendNet(zb zcode.Bytes, subnet *net.IPNet) zcode.Bytes {
```

Comment:
Regarding the existing code, not your changes: what's the reason behind the `len(subnet.Mask) == 16` check?

Reply:
`net.IPnet` has no `net.IP.To4` equivalent, so this is checking whether the mask is already a 4-byte IPv4 mask or, alternatively, a 16-byte IPv4-mapped IPv6 mask that must be trimmed to a 4-byte IPv4 mask.